azure-iot-device/tests/iothub/pipeline/test_pipeline_stages_iothub.py

```diff
@@ -322,29 +335,31 @@ def test_update_sas_token_operation(self, stage):
             stage.next.run_op.call_args[0][0], pipeline_ops_base.UpdateSasTokenOperation
         )
 
-    @pytest.mark.it(
-        "Handles any Exceptions raised by the UpdateSasTokenOperation and passes them into the unhandled exception handler"
-    )
-    def test_raises_exception(self, stage, mocker):
-        threading.current_thread().name = "not_pipeline"
+    # @pytest.mark.it(
+    #     "Handles any Exceptions raised by the UpdateSasTokenOperation and passes them into the unhandled exception handler"
+    # )
+    # def test_raises_exception(self, stage, mocker, unhandled_error_handler, arbitrary_exception):
+    #     threading.current_thread().name = "not_pipeline"
```

Comment:
will these be commented ?

Reply:
Forgot these, will fix now

---
In reply to: [332195314](https://github.com/Azure/azure-iot-sdk-python/pull/330#discussion_r332195314) [](ancestors = 332195314)